src/authentication/FlowDetails.tsx

```diff
@@ -0,0 +1,240 @@
+            provider: ex.providerId!,
+          });
+        id = result.id!;
+      }
+      const c = change as IndexChange;
+      const times = c.newIndex - c.oldIndex;
+      for (let index = 0; index < Math.abs(times); index++) {
+        if (times > 0) {
+          await adminClient.authenticationManagement.lowerPriorityExecution({
+            id,
+          });
+        } else {
+          await adminClient.authenticationManagement.raisePriorityExecution({
+            id,
+          });
+        }
+      }
+      refresh();
+      addAlert(t("updateFlowSuccess"), AlertVariant.success);
+    } catch (error) {
+      addAlert(
+        t("updateFlowError", {
+          error: error.response?.data?.errorMessage || error,
+        }),
+        AlertVariant.danger
+      );
+    }
+  };
+
+  const update = async (
+    execution: AuthenticationExecutionInfoRepresentation
+  ) => {
+    try {
+      await adminClient.authenticationManagement.updateExecution(
+        { flow: flow?.alias! },
+        execution
+      );
+      refresh();
+      addAlert(t("updateFlowSuccess"), AlertVariant.success);
+    } catch (error) {
+      addAlert(
+        t("updateFlowError", {
+          error: error.response?.data?.errorMessage || error,
+        }),
+        AlertVariant.danger
+      );
+    }
+  };
+
+  return (
+    <>
+      <ViewHeader
+        titleKey={toUpperCase(flow?.alias! || "")}
+        badges={[
+          { text: <Label>{t(usedBy)}</Label> },
+          builtIn
+            ? {
+                text: (
+                  <Label
+                    className="keycloak_authentication-section__usedby_label"
+                    icon={<CheckCircleIcon />}
+                  >
+                    {t("buildIn")}
+                  </Label>
+                ),
+                id: "builtIn",
+              }
+            : {},
+        ]}
+      />
+      <PageSection variant="light">
+        {executionList?.expandableList &&
+          executionList.expandableList.length > 0 && (
+            <Toolbar id="toolbar">
+              <ToolbarContent>
+                <ToggleGroup>
+                  <ToggleGroupItem
+                    icon={<TableIcon />}
+                    aria-label="copy icon button"
+                    buttonId="third"
+                    isSelected={tableView}
+                    onChange={() => setTableView(true)}
+                  />
+                  <ToggleGroupItem
+                    icon={<i className="fas fa-project-diagram"></i>}
+                    aria-label="undo icon button"
+                    buttonId="fourth"
+                    isSelected={!tableView}
+                    onChange={() => setTableView(false)}
+                  />
+                </ToggleGroup>
+              </ToolbarContent>
+            </Toolbar>
+          )}
+        {tableView &&
+          executionList?.expandableList &&
+          executionList.expandableList.length > 0 && (
```

Comment:
To reduce the size of this statement you could use `length` as an implicit truthy or falsy value by letting JavaScript cast it for you:

```suggestion
        {tableView && executionList?.expandableList?.length && (
```